app/views/users/show.html.erb

```diff
@@ -1 +1,15 @@
-<h1>Show a specific user</h1>
\ No newline at end of file
+<div class="user-post-main-container">
+    <%= render partial: 'users/user', locals: { user: @user } %>
+    <div class="user-bio-container">
+        <h4 class="user-post-bio">Bio</h4>
+        <p class="user-post-bio-para"><%= @user.bio %></p>
+    </div>
+    <div class="list-of-posts">
+        <% @user.three_recent_posts.each do |post| %>
+            <%= render partial: 'posts/post', locals: { post: post } %>
+        <% end %>
+    </div>
+    <a href="<%= user_posts_path(@user) %>">
+        <button class="see-all-post-btn">See all posts</button>
+    </a>
+</div>
```

Comment:
- [ ] Kindly position the post part of the user show page to match the wireframe, including the comments and likes counters.

  Your Implementation | Wireframe
  :---:|:---:
  ![image](https://github.com/nilab-khwaja/blog-app/assets/54848363/127177a9-7a9a-4225-9d18-6b250ba8eec6) | ![image](https://github.com/nilab-khwaja/blog-app/assets/54848363/8e3ff3c3-c3ec-482e-a2e2-bd2777e19ef9)